tests/integration/test_sqs.py

```diff
@@ -690,6 +690,61 @@ def test_lambda_invoked_by_sqs_message_with_delay_seconds_dotnetcore31(self):
             zip_file, handler, LAMBDA_RUNTIME_DOTNETCORE31
         )
 
+    def _run_test_fifo_queue_send_multiple_messages(self):
+        fifo_queue = 'queue-{}.fifo'.format(short_uid())
+
+        message_group = 'group-%s' % short_uid()
+        results = []
+        number_of_messages = 5
+
+        queue_url = self.client.create_queue(QueueName=fifo_queue, Attributes={'FifoQueue': 'true'})['QueueUrl']
+
+        # it should preserve .fifo in the queue name
+        self.assertIn(fifo_queue, queue_url)
+
+        # try sending multiple message with message group ID and deduplication ID
+        for i in range(number_of_messages):
+            rs = self.client.send_message(
+                QueueUrl=queue_url, MessageBody='message-{}'.format(i),
+                MessageDeduplicationId='deduplication-{}'.format(i), MessageGroupId=message_group
+            )
+            results.append(rs)
+
+        return queue_url, number_of_messages, results
+
+    def test_fifo_queue_send_multiple_messages_single_receive(self):
+        queue_url, number_of_messages, results = self._run_test_fifo_queue_send_multiple_messages()
+
+        # receive multiple message in the same time
+        messages = self.client.receive_message(
+            QueueUrl=queue_url, MessageAttributeNames=['All'], MaxNumberOfMessages=number_of_messages
```

Comment:
I just checked locally on my machine and it seems as if the test only succeeds because `MaxNumberOfMessages` is provided. Doing multiple `receive_message` without `MaxNumberOfMessages` or with `MaxNumberOfMessages` lower than `number_of_messages` still fails.

Reply:
Thanks for the update @svenallers . We have extended the test and added additional `receive_message` calls without `MaxNumberOfMessages` as well as `MaxNumberOfMessages` lower than `number_of_messages`: https://github.com/localstack/localstack/pull/2729 Unfortunately still unable - do you think you could contribute a small pull request with a test that _fails (Python code, or script with sequence of `awslocal` commands_)? Thanks for your help.

Reply:
Hey, I realized yesterday that there is probably no bug but it was my own fault why it did not work. I simply forgot to delete the message from the queue 😞. Sry for the inconvenience